ci/jest-reporter.js

```diff
@@ -1,47 +1,80 @@
+const { generateDiscordCloudwatchLogUrls, listECSTasks, sendDiscordNotification } = require('./helpers')
 const { BaseReporter } = require('@jest/reporters')
 const child_process = require('child_process')
 
+const userName = 'jest-reporter'
+let g_taskArns
```

Comment:
What does the `g_` stand for? Can we aim for more descriptive names? It will be hard for someone else coming to this code to understand what everything is otherwise.

Reply:
In my experience it has been good practice to prefix global variables with g_ for more clarity.